dts/bindings/dma/dma.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+title: DMA Base Structure
+
+description: >
+    This binding gives the base structures for all DMA devices
```

Comment:
No need to have `title:` and `description:` keys in this file, since it's not a binding in itself and is just merged into actual bindings. See e.g. `gpio-controller.yaml` for a template.